models/badges.js

```diff
@@ -27,6 +28,35 @@ const fetchBadges = async ({
   }
 }
 
+/**
+ * Fetches the data about user badges
+ * @param query { Object }: Filter for badges data
+ * @return {Promise<userBadgeModel|Array>}
+ */
+
+const fetchUserBadges = async (username) => {
+  try {
+    const userBadges = []
+    let userExists = false
+    const result = await fetchUser({ username })
+    if (result.userExists) {
+      userExists = true
+      const userID = result.user.id
+      const snapshot = await badgeModel.get()
+      snapshot.forEach((item) => {
+        if (item.data()?.users?.includes(userID)) {
+          userBadges.push({ title: item.data().title, description: item.data().description })
+        }
+      })
```

Comment:
We can improve this section of code for better readability. Do the following

- Add a line break before and after this `forEach` section
- on line no 48, this

```javascript
const { title, description } = item.data();
```

Then use this `title` & `description` like this - 
```javascript
userBadges.push({ title, description });
```

This is shorthand for object. If the key & value are same, we can use it the way I wrote. This internally means

```javascript
userBadges.push({ title:title, description:description });
```

Reply:
Okay got it, will do it.

Reply:
Regarding the line break, you mean before the `forEach` and after the `if` block right? It shows an indentation error if I directly add a line break after `forEach`.